### DISCUSSION TITLE
Anything from discussion to chatting.

### DISCUSSION THREAD

**Discussion by wb-siro** [wb-siro/THM-Translation#3] [General]
🥳

**Comment by wb-siro** [comment]
I think I'll make the Android version an official release once on Halloween🎃

**Comment by Iamg1004** [comment]
Hey! Quick question, would you like the SpellCard names (The words between the "") to remain in english, or to be translated? As far as I have seen, it has been left in english in the spanish fan patches of the official games ([https://es.touhouwiki.net/wiki/Embodiment_of_Scarlet_Devil/Spell_Cards/Nivel_2](https://es.touhouwiki.net/wiki/Embodiment_of_Scarlet_Devil/Spell_Cards/Nivel_2) for example) but either way is fine, as long as you're okay with me translating them or just leaving them be.

**Comment by wb-siro** [comment]
Hmmm... Which is better?
Seems like the clearer the better, but I'll leave it to you!

**Comment by wb-siro** [comment]
@Iamg1004 
Thanks for the translation work, It's coming along nicely!
I'd like to put your name in the staff credits, what should it be called?🤔

**Reply by Iamg1004** [reply]
Thank you! I've been trying my best :)
Ignacio Mardones G. would be fine for the credits! If you could also add my twitter (@Iamg2002) I'd really appreciate it, but I understand if you don't.

**Reply by wb-siro** [reply]
ok! I'll post your Twitter name as well!

P.S. As a token of our appreciation, I will be issuing a serial code that will allow you to perform ad-free and special action (content TBD).

**Reply by Iamg1004** [reply]
Thank you very much! It is appreciated a lot, truly.

**Reply by wb-siro** [reply]
I just sent you the code via email!
There will be more upgrades in the future, but I'd be happy to translate it if you feel like it!

**Comment by wb-siro** [comment]
@Iamg1004
Sorry!
Is there a short translation for "Configuraciones"?
The text is too small for the width of the button.😔

**Reply by Iamg1004** [reply]
Sure! No worries, I already changed it for "Ajustes", which works just as well.

**Comment by wb-siro** [comment]
@Iamg1004 
Gmail decided it was junk mail and I wasn't getting any notifications!
If you haven't received any notifications, please check your spam.

**Reply by Iamg1004** [reply]
Thankfully, I've seen all notifications in my mail, and having just checked, none have gone to my junk mail. However, I'll be watching the folder regularly, just in case. Thanks!

**Reply by wb-siro** [reply]
Thanks!
If everything's okay, it's just a normal check. 👌

**Comment by Iamg1004** [comment]
@wb-siro 
Dear しろ:
Thank you so much for the Amazon Giftcard! It's the first tangible reward I've been given for translating.
I'm sorry for not acknowledging it before, but I was in a place without any Internet for like, a week. I'm back now, and everything was translated up-to-date!
Hopefully I can keep it updated as long as it is in development, as I've been doing no problem.

Sincerely, Ignacio.

**Comment by Iamg1004** [comment]
Hi しろ!

Since you are machine translating to english, does it bother you if I make a few changes? To make it sound more natural, fix a few errors and whatnot. Mostly small stuff, so it all flows better.

**Reply by wb-siro** [reply]
Hi Ignacio! Thanks for everything!
Of course! Any improvement that makes it better!

**Reply by wb-siro** [reply]
Oh, by the way, the Chinese translation team has agreed to help us, and they want to translate the text as well.
I will add some text there, but it is very voluminous, so you don't have to force me to translate it!

**Comment by Iamg1004** [comment]
Sorry for my absence, but University has been quite hard (This is my first year going in person, and not just online). I'll keep working as hard as possible, just maybe not as fast as before. Sorry!

**Reply by Iamg1004** [reply]
(Also, what do you mean by 館でぶんぶん振り回してたけど後片付けが大変そう、これだけ大きいと避けるのも苦労するわね in Flandre's "Lävateinn" Reimu text? Translator isn't giving me anything truly useful :/)

**Reply by wb-siro** [reply]
Thanks for everything! No problem at all!
Your academics and your life come first!

~~As for the translation, it means the following!~~
~~This spell card is too big to avoid, and if you use it in the Scarlet Devil Mansion (because it's too big and will destroy the whole area), it will be hard to clean up afterwards!~~

**Reply by wb-siro** [reply]
Sorry! My mistake!
I changed it to another sentence, so please check again🙏
